tests/phpunit/util/class-util.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+class Util {
+
+    static public function has_value ( $value, $array ) {
+        $has_value = false;
+
+        $callback = function  ( $v, $k ) use( $value, &$has_value ) {
```

Comment:
remove the spaces before 1st parameter and after 2nd parameter.
follow php coding guidelines and update the code